### PR TITLE
Chaging elections list to reflect current officers.

### PR DESCRIPTION
**Pull request by jberkus**
New list reflects current officers: me, Jaice DuMars, Ihor.

Amended and force pushed to get correct list.

### CODE REVIEW

File: groups/groups.yaml [hunk 1]
```diff
@@ -183,10 +183,10 @@ groups:
       WhoCanPostMessage: "ANYONE_CAN_POST"
       ReconcileMembers: "true"
     owners:
-      - briangrant@google.com
       - ihor@cncf.io
-      - jorgec@vmware.com
+      - josh@berkus.org
       - killen.bob@gmail.com
```
Comment (cblecker) [inline groups/groups.yaml:188]: Bob should be removed

Comment (jberkus) [reply groups/groups.yaml:188]: so just me/ihor/jaice?  OK.  Wasn't clear.

Comment (jberkus) [reply groups/groups.yaml]: Fixed and squash-pushed

**Comment by dims** [comment]
@jberkus @jdumars feel free to remove hold when ready

**Pull request merged**